WeatherDAL/IWeather.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WeatherDAL
+{
+    public interface IWeather
+    {
+        string CurrentWeather(string url, string appid, string city);
```

Comment:
Use verb for method naming. GetCurrentWeather

Reply:
string url, string appid, move to Console app config, and them should be passed through the class constructor

Reply:
Create WetherProvider class in dal